### DISCUSSION TITLE
Excluding

### DISCUSSION THREAD

**Discussion by Ethan-Bit-256** [premake/premake-core#1915] [Q&A] [answered]
How do I exclude files/folder from my project?

**Comment by samsinsane** [comment, marked as answer]
Have tried searching the website for [this](https://premake.github.io/docs/Adding-Source-Files/#excluding-files)?

**Reply by Ethan-Bit-256** [reply]
Thanks! AND yes, I tried, but my key words didn't work on the search bar.

**Reply by Ethan-Bit-256** [reply]
And it worked! woohooo!
